.github/workflows/compute.yml

```
@@ -0,0 +1,56 @@
+      command:
+        description: "Command from the Kernel"
+        required: false
+
+jobs:
+  compute:
+    name: "plugin name"
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      contents: read
+      labels: write
```

actionlint (1.7.12)

[error] unknown permission scope "labels". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)